app/scripts/views/sign_up.js

```diff
@@ -314,6 +315,7 @@ function (Cocktail, p, BaseView, FormView, Template, AuthErrors, mailcheck,
 
           if (preVerifyToken && account.get('verified')) {
             self.logViewEvent('preverified.success');
+            self.onSignInSuccess(account);
```

Comment:
Ah, I see, you are trying to avoid adding more function calls too `views/sign_in` and `views/force_auth`. This `onSignInSuccess` wraps their `onSignInSuccess` methods.

More thoughts incoming.